<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="TreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>821</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>90</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>260</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="addBox">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>40</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="delBox">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>210</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="TreeView" name="treeView_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>971</width>
      <height>821</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="delBox_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>490</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="addBox_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>320</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>540</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>370</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTextEdit</extends>
   <header>treeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
